framework/contracts/native/ibc-host/tests/integration.rs

```diff
@@ -189,3 +214,66 @@ fn execute_action_with_account_creation() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn execute_send_all_back_action() -> anyhow::Result<()> {
+    let sender = Addr::unchecked("sender");
+    let chain = Mock::new(&sender);
+
+    let admin = Addr::unchecked("admin");
+    let mut admin_chain = chain.clone();
+    admin_chain.set_sender(admin.clone());
+
+    let admin_abstr = Abstract::deploy_on(admin_chain.clone(), admin.to_string())?;
+    let abstr = Abstract::load_from(chain.clone())?;
+
+    let account_sequence = 1;
+    let chain = "juno";
+
+    // We need to set the sender as the proxy for juno chain
+    admin_abstr
+        .ibc
+        .host
+        .register_chain_proxy(chain.into(), sender.to_string())?;
```

Comment:
I think it's a bit confusing to use the sender as the proxy in this test. 

Can you add a `polytone_proxy` address and use the `abstr.ibc.host.call_as(polytone_proxy)...` syntax below?